File: 5-Patron/php/Particulas.php
<?php
class Particulas{
	private $id;
	private $archivo;

	public function __construct($id, $archivo){
    $this->id = $id;
    $this->archivo = $archivo;
	}

	public function getId(){
		return $this->id;	
	}

	public function setId($id){
		$this->id = $id;
	}

	public function getArchivo(){
		return $this->archivo;	
	}

	public function setArchivo($archivo){
		$this->archivo = $archivo;
	}

	public function cargarParticulas(){
		$ruta = 'C:\CyberMagik2069\Particles\Particulas.txt';
		echo 'Cargando archivo...';
		echo $ruta . ' detectada.';
	}
}
?>

File: 5-Patron/Java/Particulas.java
package patron5;

import java.io.File;

public class Particulas {
    private int id;
    private File file;

    public Particulas(int id, File file) {
        this.id = id;
        this.file = file;
    }

    public File getFile() {
        return file;
    }

    public int getId() {
        return id;
    }

    public void setFile(File file) {
        this.file = file;
    }

    public void setId(int id) {
        this.id = id;
    }
    
    public void cargarParticulas(File file){
        String ruta = "C:\\CyberMagik2069\\Particles\\Particulas.txt";
        System.out.println("\nCargando archivos....\n");
        System.out.println(ruta + " Detectada.");
    }
}
